saleor/webhook/transport/synchronous/circuit_breaker/breaker_board.py

```diff
@@ -41,12 +89,21 @@ def register_error(self, app_id: int):
             f"{0}-{1}".format(app_id, "error"), self.ttl
         )
 
-        if errors >= self.failure_threshold:
+        open = False
+        if self.mode == BreakerMode.FIXED and errors >= self.failure_threshold:
+            open = True
+        elif self.mode == BreakerMode.PERCENTAGE:
+            total = self.storage.get_event_count("success") + errors
+            if errors * 100 / total >= self.failure_threshold:
```

Comment:
(Nitpick) This is equivalent but much more easier to understand by my brain: `(errors / total) * 100`.